playbook/app/pb_kits/playbook/pb_legend/legend.rb

```diff
@@ -4,7 +4,7 @@ module Playbook
   module PbLegend
     class Legend < Playbook::KitBase
       prop :color, type: Playbook::Props::Enum,
-                   values: (1..7).map { |n| "data_#{n}" },
+                   values: %w[data_1 data_2 data_3 data_4 data_5 data_6 data_7 success warning error info neutral primary],
```

Comment:
there is a strong argument here that these values are [magical](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad#:~:text=In%20programming%2C%20a%20%22magic%20number,in%20more%20than%20one%20place). It may be prudent to extract them into a constant, which (at least on the rails side), can then be referenced in the two places we use it. On the react side, I believe there is a way to define custom datatypes, though I'm not sure whether that precedent has been set in Playbook.

Reply:
So I followed the pattern for defining prop values in Playbook. I can extract it and in a constant if that's what you meant.